.github/workflows/unit-tests.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         if: always() && steps.build-babelfishpg_unit.outcome == 'success'
         timeout-minutes: 60
         run: |
-          sudo ~/psql/bin/psql -d jdbc_testdb -U runner -f .github/scripts/unit_tests.sql > >(tee ~/psql/output.out) 
+          sudo ~/psql/bin/psql -v ON_ERROR_STOP=1 -d jdbc_testdb -U runner -f .github/scripts/unit_tests.sql > >(tee ~/psql/output.out) 2>&1
```

Comment:
My original comment was to do this at all other places as well, i.e. where we are creating tsql/common/tds extensions. But it's fine if you want to limit the scope.

Reply:
Updated in almost all occurrences.